nodeJS/apis/api_security.md

```diff
@@ -22,14 +22,16 @@ This strategy, while particularly useful with APIs can be used with a traditiona
 <div class="lesson-content__panel" markdown="1">
 
 1. [This video](https://www.youtube.com/watch?v=7nafaH9SddU) is a great resource that explains everything you need to know about creating and verifying JSON Web Tokens.
-2. [This article](https://dev.to/_arpy/learn-using-jwt-with-passport-authentication-22n8) covers setting up JSON Web Tokens within the PassportJS system that you should already be familiar with. In this case you don't _really_ need Passport, but it does handle quite a bit of stuff that the guy in the video set up manually.
+2. [This article](https://dev.to/_arpy/learn-using-jwt-with-passport-authentication-22n8) covers setting up JSON Web Tokens within the PassportJS system that you should already be familiar with. In this case you don't _really_ need Passport, but it does handle quite a bit of stuff that the guy in the video set up manually. NOTE: There is some typos in the code blocks provided in the article, its good exercise to code along and spot them ;). Aswell, some function are now deprecated and this needs some extra reading of documentation in order to make it work.
+3. [This video](https://www.youtube.com/watch?v=JdGOb7AxUo0) is a good counterbalance in the use of JWT versus use of Session Cookie. It states some facts and is a good way to comprehend a larger scope of the use of JWT for authentication and some of the danger coming along.
```

Comment:
oof... i'm not sure we need to add a whole hour-long vid here just to explain the difference between JWT and Session cookies.

This would fit better in 'additional resources'

also, your wording here is a little awkward... i think it's more accurate to say something like:

> Not everyone agrees that JWTs are the best way to store authentication data. [This video](link here) is one argument against using them, and some of the pitfalls that you may encounter with using them.

Reply:
Thank you for your wording! I've updated following your recommandations